Source/WebKit/WebProcess/InjectedBundle/API/c/WKBundlePage.cpp

```diff
@@ -287,7 +287,10 @@ void* WKAccessibilityFocusedObject(WKBundlePageRef pageRef)
     if (!page)
         return 0;
 
-    RefPtr focusedDocument = CheckedRef(page->focusController())->focusedOrMainFrame().document();
+    RefPtr focusedOrMainFrame = CheckedRef(page->focusController())->focusedOrMainFrame();
+    if (!focusedOrMainFrame)
+        return 0;
```

Comment:
Since you're touching this function, some `nullptr` conversion would have been nice.